src/test/java/org/takes/facets/auth/social/PsLinkedinTest.java

```diff
@@ -67,100 +60,271 @@
      */
     @Test
     public void logins() throws IOException {
+        final String tokenpath = "/uas/oauth2/accessToken";
+        final String peoplepath = "/v1/people";
```

Comment:
@mkordas I think we can, but I thought that since `tokenpath` needs to be declared and passed to the inner class, the `peoplepath` should be declared too, for symmetry :)

I will change it.